README.MD

```diff
@@ -50,13 +54,50 @@ const result = pm.validate('MyPassword123'); // { valid: false, messages: ['must
 
 ```
 
-### Validators
+2. `fluent` - This allows you chain multiple validators together on the `PasswordValidatorManager` in a more functional programming style.
+This is useful when you want to validate a password without having to register the validators with the manager. It still uses the same validators as the `standard` API but provides an interface for chaining them together.
+
+
+```ts
+import { PasswordValidatorManager } from '@password-validator/core';
+
+const result = PasswordValidatorManager.fluent()
+  .min(6) // Minimum length of 6 characters
+  .digit(1) // At least 1 digit
+  .specialCharacter(1) // At least 1 special character
+  .validate("eihi2kd#"); // { valid: true, messages: [] } --> Password is valid
+```
+
+NB: Ensure to call the `validate` method with the `password` to be validated. Without this, the password validation will not be triggered.
+
+
+## Validators
 
-A validator is class that can validate a password and return a result. Validators should typically be registered with the `PasswordValidatorManager` (as shown above). The library comes with a number of validators (see table below) that can be used to build a validation suite.
+A validator is class that can validate a password and return a result. This is used by both the `Standard` and the `Fluent` API. The library comes with a number of validators (see table below) that can be used to build a validation suite.
 
-Each validator receives a `passwordRule` _(number)_ argument in its constructor which defines the number of expected characters associated with the validator. For example, the `const minLength = new MinLengthValidator(10)` defined above means that the passowrd must be atleast __10 characters__ long to be considered valid.
+Each validator receives a `passwordRule` _(number)_ argument, which defines the number of expected characters associated with the validator.
+Consider the following example:
 
-NB: The only exception to this is the `NoSpaceCharacterValidator` which requires no arguments and simply ensures the password contains no spaces.
+```ts
+// Standard API
+import { MinLengthValidator } from '@password-validator/core';
+
+const minLength = new MinLengthValidator(10);
+const result = minLength.validate("eihi2kd#");
```

Comment:
Remember that the users don't need to use the validators directly except through the `PasswordValidatorManager` class. Please update the usage sample to something like below:

```ts
import { PasswordValidatorManager } from '@password-validator/core';

const pm = PasswordValidatorManager.standard();

const minLength = new MinLengthValidator(10);
pm.register(minLength);

const result = pm.validate("eihi2kd#");
```

Reply:
I probably did this with sleep in my eyes. Apologies 😄

Reply:
scratch that! this actually uses pasword  manager: `const result = pm.validate("eihi2kd#")`, it's just using one validator. 

I wanted to use just one validator to explain `passwordRule` and its usage.

Reply:
I understand but you can still do that with the password validator manager